packages/terra-clinical-item-collection/src/Item.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ItemView from 'terra-clinical-item-view';
+import List from 'terra-list';
+import Table from 'terra-table';
+import 'terra-base/lib/baseStyles';
+import styles from './ItemCollection.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The display elements to be presented.
+   */
+  children: PropTypes.node,
+  /**
+   * The start accessory element to be presented.
+   */
+  startAccessory: PropTypes.node,
```

Comment:
`startAccessory`, `comment`, and `endAccessory` should be of type element in my option to reflect the ItemView API. The reason I chose type node was that the [TableCell component does not allow empty content](https://github.com/cerner/terra-core/blob/master/packages/terra-table/src/TableCell.jsx#L9). When creating the table displays, this meant I would need to supply the content with a filler element or an empty string. An empty string seemed more appropriate & type node allows for this string, but resulted in wrapping the `startAccessory`, `comment`, and `endAccessory` in a div to ensure they are an element.

@bjankord @mjhenkes Do we remember why TableCell requires content? Given the component type, it seems there was a good reason since I'd expected empty cells in a table.

Reply:
Updating ItemView's props to accept nodes in #222